src/font/lv_font.h

```diff
@@ -28,6 +28,7 @@ extern "C" {
 
 /* imgfont identifier */
 #define LV_IMGFONT_BPP 9
+#define LV_VECFONT_BPP 0
```

Comment:
Using the BPP for marking format is hacky. To make it more generic we should rename `font->bpp` to `font->glyph_format`. 
Let's make it an `enum` like:
```c
typedef enum {
  LV_FONT_GYLPH_FORMAT_A1 = 1,
  LV_FONT_GYLPH_FORMAT_A2 = 2,
  LV_FONT_GYLPH_FORMAT_A4 = 4,
  LV_FONT_GYLPH_FORMAT_A8 = 8,
  LV_FONT_GYLPH_FORMAT_VECTOR = 9,
  LV_FONT_GYLPH_FORMAT_IMAGE = 10,
}lv_font_glyph_format_t;
```

It required to update font converter but we are still in time to do that.

Reply:
OK, it's great!

I think we can create a new PR to handle this font format things to make it more clear.